_src/static/sass/utilities/_variables.scss

```diff
@@ -3,156 +3,316 @@
 
 // About: Sass partial for defining variables used throughout the applications.
 
-// #Grid:                    Grid and layout variables
-// #Fonts:                   Font definitions
-// #Colors (Palette):        Base and theme color definitions
-// #Colors (Application):    Colors for use within the entire application
-// #Timing:                  Event timing variables
-// ----------------------------
+// #GRID:           Grid and layout variables
+// #COLORS:         Base, palette and theme color definitions + application
+// #TYPOGRAPHY:     Font definitions and scales
+// #DEPTH:          Event timing variables
+// #TIMING:         Event timing variables
+// #APPLICATIONS:   Applying configuration
 
 
 // ----------------------------
-// #Grid
+// #GRID
 // ----------------------------
-$baseline:     rem(20);
-
-$columns:      12;
-$min-width:    rem(768);
-$max-width:    rem(1600);
-
+// grid - baseline
+$baseline:      rem(20);
+$baseline-v:    ($baseline*2);
+$baseline-h:    $baseline;
 
 // ----------------------------
-// #Fonts
+// #COLORS
 // ----------------------------
-$font-sans:     'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-serif:    'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
-$font-size:     1rem;
+// color: config
+$transparent:       hsla(0%, 0%, 0%, 0.0);
 
+$gray:              hsla(0%, 0%, 49%, 1.0);
+$red:               hsla(16%, 52%, 36%, 1.0);
+$yellow:            hsla(4%, 83%, 58%, 1.0);
+$green:             hsla(39%, 50%, 43%, 1.0);
+$blue:              hsla(55%, 82%, 45%, 1.0);
+$pink:              hsla(16%, 50%, 43%, 1.0);
+
+$brand-primary:     $blue;
+$brand-secondary:   $pink;
+$brand-accent:      $gray;
+
+// colors: palettes
+$palettes: (
+    primary:        (
+        base:       $brand-primary,
```

Comment:
Good question - This is creating monochromatic scales of values per color (the actual color base is  defined/abstracted in good old Sass variables beforehand, for easier configuration and theming). I'm fine with pairing down the computed scale once we start using them, but this gives us different shades of each base color (primary, secondary, accent, black-white scale, and system colors like red, green, yellow) which seems barebones enough.